assets/migrations/sqlite/001_initialize_schema.sql

```diff
@@ -0,0 +1,56 @@
+-- +goose Up
+CREATE TABLE tuple (
+    store CHAR(26) NOT NULL,
+    object_type VARCHAR(128) NOT NULL,
```

Comment:
As far as I understand SQLite, when defining something like `VARCHAR(255)` everything in parentheses is ignored and it will basically be a `TEXT` column. It’s interesting here to know, that a length of 255 is to be expected, but I would simply make it a `TEXT` and eventually add a `CHECK` constraint to check the length, if one can’t guarantee it from OpenFGA.

Reply:
Interesting, I have not experimented with CHECK constraints in SQLite. In general I'd tend toward checking length within OpenFGA itself if needed since the behavior of MySQL for example can vary according to the configuration (truncate and warn for non-strict mode, error in strict mode).